features/machine/machine.feature

```diff
@@ -370,28 +370,50 @@ Feature: Machine features testing
     And I switch to cluster admin pseudo user
     And I use the "openshift-machine-api" project
     Then admin ensures machine number is restored after scenario
 
     Given I store the last provisioned machine in the :machine clipboard
     Then evaluation of `machine(cb.machine).azure_location` is stored in the :default_location clipboard
-    And admin ensures "default-valued-33058" machineset is deleted after scenario
+    And admin ensures "<name>" machineset is deleted after scenario
 
-    Given I obtain test data file "cloud/ms-azure/ms_default_values.yaml"
-    When I run oc create over "ms_default_values.yaml" replacing paths:
-      | n                                                                                         | openshift-machine-api                           |
-      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]           | <%= infrastructure("cluster").infra_name %>     |
-      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"]        | default-valued-33058                            |
-      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-cluster"]    | <%= infrastructure("cluster").infra_name %>     |
-      | ["spec"]["template"]["spec"]["providerSpec"]["value"]["location"]                         | <%= cb.default_location %>                      |
-      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-machineset"] | default-valued-33058                            |
+    Given I obtain test data file "cloud/ms-azure/<file_name>"
+    When I run oc create over "<file_name>" replacing paths:
+      | n                                                                                         | openshift-machine-api                       |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]           | <%= infrastructure("cluster").infra_name %> |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"]        | <name>                                      |
+      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-cluster"]    | <%= infrastructure("cluster").infra_name %> |
+      | ["spec"]["template"]["spec"]["providerSpec"]["value"]["location"]                         | <%= cb.default_location %>                  |
+      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-machineset"] | <name>                                      |
+      | ["metadata"]["name"]                                                                      | <name>                                      |
     Then the step should succeed
+  
+    And I wait up to 120 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource | machine |
+    Then the step should succeed
+    And the output should contain:
+      | Running |
+    """
```

Comment:
These steps always work, as no machine name is specified, we can use below steps to get the same result after  line 405.
```
    And I wait up to 120 seconds for the steps to pass:
    """
    Then the expression should be true> machine(cb.machine_latest).phase(cached: false) == "Running"
    """
```

Reply:
Agreed , will need this fix for one more test  , will do it with a seperate branch , this one is corrected